soc/wch/ch32v/qingke_v2a/Kconfig.defconfig

```diff
@@ -1,20 +1,11 @@
-# Copyright (c) 2024 Michael Hope
+# Copyright (c) 2024-2025 Michael Hope
 # SPDX-License-Identifier: Apache-2.0
 
 if SOC_SERIES_QINGKE_V2A
 
 config SYS_CLOCK_HW_CYCLES_PER_SEC
 	default $(dt_node_int_prop_int,/cpus/cpu@0,clock-frequency)
 
-config MAIN_STACK_SIZE
```

Comment:
These are needed on the CH32V003, which is the only SOC using the V2A.

Reply:
Please look at the new version while i run functionaility tests.

Reply:
fixed